docs/gallery/domain/images.py

```diff
@@ -245,6 +245,58 @@
 nwbfile.add_acquisition(images)
 
 ####################
+# IndexSeries for repeated images
+# -------------------------------
+#
+# You may want to set up a time series of images where some images are repeated many
+# times. You could create an :py:class:`~pynwb.image.ImageSeries` that repeats the data
+# each time the image is shown, but that would be inefficient, becuase it would store
+# the same data multiple times. A better solution would be to store the unique images
+# once and reference those images. This is how :py:class:`~pynwb.image.IndexSeries`
+# works. First, create an :py:class:`~pynwb.base.Images` container with the order of
+# images defined using a :py:class:`~pynwb.base.ImageReferences`. Then create an
+# :py:class:`~pynwb.image.IndexSeries` that indexes into the
+# :py:class:`~pynwb.base.Images`.
+
+from scipy import misc
+
+from pynwb.image import Images, IndexSeries, GrayscaleImage, RGBImage
+from pynwb.base import ImageReferences
+
+
+gs_face = GrayscaleImage(
+    name="gs_face",
+    data=misc.face(gray=True),
+    description="Grayscale version of a raccoon.",
+    resolution=35.433071,
+)
+
+rgb_face = RGBImage(
+    name="rgb_face",
+    data=misc.face(),
+    resolution=70.0,
+    description="RGB version of a raccoon.",
+)
+
+images = Images(
+    name="raccoons",
+    images=[rgb_face, gs_face],
+    description="A collection of raccoons.",
+    order_of_images=ImageReferences("order_of_images", [rgb_face, gs_face]),
+)
+
+idx_series = IndexSeries(
+    name="stimuli",
+    data=[0, 1, 0, 1],
+    indexed_images=images,
+    unit="N/A",
```

Comment:
Couple things here;

1) should this be `n.a.`? I think we've usually followed that standard (also that's how other units in this gallery are specified)

2) For something called an `IndexSeries`, it seems so weird to even have units at all, and feels weirder to even be required to specify it. What would you think about removing it or at least relaxing it to be optional (and default to None or something)?

Reply:
Yeah, actually you need to specify it. It's a backwards compatibility quark: https://github.com/NeurodataWithoutBorders/pynwb/blob/e03b0a932c4f83c9f143d4a96973e52219ce7917/src/pynwb/image.py#L234-L236

Reply:
Hmm wow OK lol